src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs

```diff
@@ -30,5 +30,130 @@ public static class Claims
             public const string Specialties = "urn:linkedin:specialties";
             public const string Summary = "urn:linkedin:summary";
         }
+
+        public const string EmailAddressField = "email-address";
```

Comment:
I'm probably missing some nuance, but what's the difference between this one being here and the others being in the subclass?

Reply:
See https://developer.linkedin.com/docs/fields/basic-profile#
It just belongs to the other group